// Copyright 2017 Google Inc.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//    http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto3";

package scanner;

service Scanner {
  rpc Ping(PingRequest) returns (PingReply) {}

  rpc Run(RunRequest) returns (RunReply) {}

}

message RunRequest {
	string config_dir = 1;
}

message RunReply {
	int32 status = 1;
}

message PingRequest {
  string data = 1;
}

message PingReply {
  string data = 1;
}

